content/api/commands/session.md

```diff
@@ -0,0 +1,451 @@
+---
+title: session
+---
+
+Cache and restore [cookies](/api/cypress-api/cookies),
+[`localStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage),
+and
+[`sessionStorage`](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)
+in order to reduce test setup times.
+
+<Alert type="warning">
+<p>The <code>cy.session</code> API is currently experimental, and can be enabled by
+  setting the <code>experimentalSessionSupport</code> flag to <code>true</code>
+  in the Cypress config or by using <code><a href="/api/cypress-api/config">Cypress.config(...)</a></code>
+  at the top of a spec file.</p>
+<p>Enabling this flag does two things:</p>
+<ul>
+  <li>It enables the <code>cy.session</code> API for use in tests.
+  <li>It adds a new behavior (that will be the default in a future major update
+  of Cypress), where the page is cleared (by setting it to <code>about:blank</code>)
+  and all stored session data (cookies, <code>localStorage</code> and
+  <code>sessionStorage</code>) are cleared at the beginning of each test.</li>
+</ul>
+<p>You must explicitly call call <code>cy.visit</code> in each test to visit a page
+in your application.</p>
```

Comment:
You shouldn't need to use HTML.

Reply:
I did some digging, and apparently as long as you keep an empty line between HTML/MDX tags and the content, it will format the content as markdown. I'll fix this in the next commit.

Reply:
Fixed in d474bf0